feature-group-definitions/bigint.dist.yml

```diff
@@ -26,3 +26,11 @@ compat_features:
   - javascript.builtins.BigInt.toLocaleString
   - javascript.builtins.BigInt.toString
   - javascript.builtins.BigInt.valueOf
+  - javascript.builtins.BigInt64Array
+  - javascript.builtins.BigInt64Array.BigInt64Array
+  - javascript.builtins.BigUint64Array
+  - javascript.builtins.BigUint64Array.BigUint64Array
+  - javascript.builtins.DataView.getBigInt64
+  - javascript.builtins.DataView.getBigUint64
+  - javascript.builtins.DataView.setBigInt64
+  - javascript.builtins.DataView.setBigUint64
```

Comment:
@foolip I saw that you reviewed https://github.com/mdn/browser-compat-data/pull/22211, which makes this change. Did you review this specifically? I'm not 100% certain that this should be one feature (I could see `BigInt`-typed arrays as a distinct feature) and I wanted to get your take.

Reply:
I didn't really look past the name in https://github.com/mdn/browser-compat-data/pull/22211. Having taken a look, I think we should treat this more as part of typed arrays than as BigInt. These typed arrays don't store arbitrary BigInts, they store signed or unsigned 64-bit integers, and BigInt is the type you have to use to represent that.

Reply:
Oh sorry, I thought I had limited my PR to just adding snapshots. Must have missed these when rebasing.

Reply:
@Elchi3 can you isolate those changes into a revert PR so that https://github.com/mdn/browser-compat-data/pull/22211 plus that revert PR together only adds snapshot tags?

Reply:
@ddbeck I think we should comment out those keys so that we can still upgrade BCD.

Reply:
Submitted https://github.com/mdn/browser-compat-data/pull/22869

Reply:
Done with 395a20128cf8f4ea356b999807bd3edbbeffb716.